news/2023/2023-11-16-osu-world-cup-2023-quarterfinals-recap.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "osu! World Cup 2023: Quarterfinals Recap"
+date: 2023-11-16 12:00:00 +0000
+---
+
+Tune in to catch up on the latest upsets and heartbreaks (of which there were many), and prepare yourself for the upcoming semifinals weekend.
```

Comment:
> latest upsets and heartbreaks

i feel like it might be better/more balanced to have a positive/neutral + negative term rather than 2 consecutive negative ones

Reply:
[Discussed here](https://discord.com/channels/188630481301012481/218677502141399041/1174704027037077504) I will leave open if anyone wants to further discuss this or suggest a change

Reply:
i think this is pretty ok as is